README.md

```diff
@@ -39,7 +39,79 @@ In your application.js:
 ```
 ## Usage
 
-We're not sure yet.
+Given a simple application with Tasks belonging to Projects, the first thing to do is allow your Project model to accepts_nested_attributes_for.
+
+**Models**
+
+```ruby
+class Project < ApplicationRecord
+  has_many :tasks, inverse_of: :project
+  accepts_nested_attributes_for :tasks, reject_if: :all_blank, allow_destroy: true
+end
+
```

Comment:
Add a comment here
`# models/task.rb`

Reply:
seems redundant, I don't thinks its needed considering Cocoon doesn't do it.

Reply:
I did it though.